pilot/pkg/xds/ads.go

```diff
@@ -234,6 +234,12 @@ func (s *DiscoveryServer) processRequest(req *discovery.DiscoveryRequest, con *C
 
 	request.Reason = append(request.Reason, model.ProxyRequest)
 	request.Start = time.Now()
+	// SidecarScope for the proxy may has not been updated based on this pushContext.
```

Comment:
/proxy may has not been updated -> proxy may not have been updated

Reply:
thanks for comment btw, it is really useful

Reply:
Done